engine/servermaster/jobmanager.go

```diff
@@ -313,6 +334,25 @@ func (jm *JobManagerImpl) ListJobs(ctx context.Context, req *pb.ListJobsRequest)
 	return resp, nil
 }
 
+// setDetailToMasterMeta sets the results from GetJobDetail to master meta
+func setDetailToMasterMeta(masterMeta *frameModel.MasterMetaKVData, detail []byte, errJob error) {
+	if errJob != nil {
+		// Currently, we simply ignore 404 error
+		if errors.ErrorEqual(errJob, derrors.ErrJobManagerRespStatusCode404) {
+			log.Warn("get job detail from jobmaster fail", zap.Error(errJob))
+			return
+		}
+
+		// TODO: deal the response body here after we has normalized the error response format
+		log.Error("get job detail from jobmaster fail", zap.Error(errJob))
+		// TODO: we should not put the error message here directly
+		masterMeta.ErrorMsg = errJob.Error()
+	} else if detail != nil {
+		log.Debug("get job detail from jobmaster", zap.Any("detail", detail))
```

Comment:
```suggestion
```
Not necessary, we have the same debug log in job http client.

Reply:
Fix it.